parser/src/main/java/me/vertretungsplan/parser/LegionBoardParser.java

```diff
@@ -0,0 +1,347 @@
+/*
+ * substitution-schedule-parser - Java library for parsing schools' substitution schedules
+ * Copyright (c) 2016 Johan v. Forstner
+ * Copyright (c) 2016 Nico Alt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+ * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package me.vertretungsplan.parser;
+
+import me.vertretungsplan.exception.CredentialInvalidException;
+import me.vertretungsplan.objects.credential.Credential;
+import me.vertretungsplan.objects.credential.UserPasswordCredential;
+import me.vertretungsplan.objects.Substitution;
+import me.vertretungsplan.objects.SubstitutionSchedule;
+import me.vertretungsplan.objects.SubstitutionScheduleData;
+import me.vertretungsplan.objects.SubstitutionScheduleDay;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.http.client.HttpResponseException;
+import org.joda.time.LocalDate;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.nodes.TextNode;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
```

Comment:
As I use Geany for programming (that's like Notepad on Windows with syntax highlighting), I cannot do this easily. Can you tell me which imports are unused or do it yourself after merging?